governance/program/src/state/realm_config.rs

```diff
@@ -10,6 +13,91 @@ use spl_governance_tools::account::{get_account_data, AccountMaxSize};
 
 use crate::{error::GovernanceError, state::enums::GovernanceAccountType};
 
+use crate::state::realm::GoverningTokenConfigArgs;
+
+use crate::state::realm::{RealmConfigArgs, RealmV2};
+
+/// The type of the governing token defines:
+/// 1) Who retains the authority over deposited tokens
+/// 2) Which token instructions Deposit, Withdraw and Revoke (burn) are allowed
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub enum GoverningTokenType {
+    /// Liquid token is a token which is fully liquid and the token owner retains full authority over it
+    /// Deposit - yes
+    /// Withdraw - yes  
+    /// Revoke - no, Realm authority cannot revoke liquid tokens
+    Liquid,
+
+    /// Membership token is a token controlled by Realm authority
+    /// Deposit - yes, membership tokens can be deposited to gain governance power
+    /// Withdraw - no, after membership tokens are deposited they are no longer transferable and can't be withdrawn
+    /// Revoke - yes, Realm authority can Revoke (burn) membership tokens
+    Membership,
+
+    /// Dormant token is a token which is only a placeholder and its deposits are not accepted and not used for governance power within the Realm
+    /// Note: When an external voter weight plugin which takes deposits of the token is used then the type should be set to Dormant
+    /// Deposit - no, dormant tokens can't be deposited into the Realm
+    /// Withdraw - yes, tokens can still be withdrawn from Realm to support scenario where the config is changed while some tokens are still deposited
+    /// Revoke - no, Realm authority cannot revoke dormant tokens
+    Dormant,
```

Comment:
It has _'for future release'_ semantics for some DAOs, and indicates as not being used yet. For example a `Multisig` team DAO which starts as a multisig but  has the intention to eventually decentralise and start using the `Community` token.

At the same time the token deposits are not used for other `DAOs` (when a plugin taking deposits is enabled) but the token as identity of the voting population is still in use and `Unused` feels bit confusing. 

I was also considering `Proxy` or `Placeholder` but was not sure about it. Another option would be to add a 4th type ex. `PluginManaged` which would have exactly the same semantics as `Dormant` but would explicitly state the intent. What do you think?

Reply:
Ah actually, in that case `Dormant` is a great word.  It might be nice to also have `PluginManaged` so you know right away, but if you don't need it yet, you can always rename it later